packages/sheets/src/basics/interfaces/selection-config.ts

```diff
@@ -24,3 +24,8 @@ export interface ISelectionConfig {
 export interface ISelectionsConfig {
     [subUnitId: string]: ISelectionConfig[];
 }
+
+export interface IDiscreteRange {
```

Comment:
The name is impropriate because they are not ranges but row and col numbers. Could you please change it? And this interface is declared twice. Please remove the other one.